tests/linkcommand.js

```diff
@@ -324,25 +332,33 @@ describe( 'LinkCommand', () => {
 
 				expect( getData( model ) ).to.equal( 'foo[<$text linkHref="url">url</$text>]bar' );
 			} );
+
+			it( 'should add additional attributes to link if those are falsy but decorator\'s defaultValue is set to true', () => {
```

Comment:
Why storing the `false` value in the model? Also, how do you actually remove the value from the model?

Reply:
As I see this – the only thing that needs to change when `defalutValue` is `true` is the state of this button in the UI when you create a new link. Since it'll be on at that point, the command will be executed with `isSth:true` and then our job's done. The rest will work the way it works so far. Do I miss something?

Reply:
As discussed on a call:
* `LinkFormView` is binding its `isOn` property to `value` of manual decorator,
* `value` of a manual decorator is updated every time `LinkCommand` is `refresh`ed,
* every time `value` is extracted from the model's attribute,
* we need to store `false` in the attribute to indicate that user already decided whether the manual decorator is supposed to be applied or not